src/www/interfaces.php

```diff
@@ -1805,25 +1806,33 @@ function toggle_allcfg() {
                     </thead>
                     <tbody>
                       <tr>
-                        <td><i class="fa fa-info-circle text-muted"></i> <?= gettext('Enable') ?></td>
-                        <td>
+                        <td style="width:22%"><i class="fa fa-info-circle text-muted"></i> <?= gettext('Enable') ?></td>
+                        <td style="width:78%">
                           <input id="enable" name="enable" type="checkbox" value="yes" <?=!empty($pconfig['enable']) ? 'checked="checked"' : '' ?>/>
                           <strong><?= gettext('Enable Interface') ?></strong>
                         </td>
                       </tr>
                       <tr>
-                        <td><i class="fa fa-info-circle text-muted"></i> <?= gettext('Lock') ?></td>
-                        <td>
+                        <td style="width:22%"><i class="fa fa-info-circle text-muted"></i> <?= gettext('Lock') ?></td>
+                        <td style="width:78%">
                           <input id="lock" name="lock" type="checkbox" value="yes" <?=!empty($pconfig['lock']) ? 'checked="checked"' : '' ?>/>
                           <strong><?= gettext('Prevent interface removal') ?></strong>
                         </td>
                       </tr>
+                      <tr>
+                        <td style="width:22%"><a id="help_for_descr" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Description"); ?></td>
+                        <td style="width:78%">
+                          <input name="descr" type="text" id="descr" value="<?=$pconfig['descr'];?>" />
+                          <div class="hidden" data-for="help_for_descr">
+                            <?= gettext("Enter a description (name) for the interface here."); ?>
+                          </div>
+                        </td>
+                      </tr>
                     </tbody>
                   </table>
                 </div>
               </div>
-              <div id="allcfg" style="display:none">
```

Comment:
embedding dislay:none is an intentional mobile optimisation

Reply:
now retained